docs/destinations/forwarding-events/snowbridge/concepts/scaling/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Scaling"
+date: "2023-02-27"
+sidebar_position: 150
+---
+
+# Scaling
+
+## Horizontal scaling
+
+Snowbridge is built to suit a horizontal scaling model, and you can safely deploy multiple instances of Snowbridge to consume the same input out-of-the-box (the kinesis source requires Dynamo DB tables to be created for this purpose). No addditional configuration or setup is required for the app to smoothly run across multiple instance/environment, compared to a single instance/environment.
+
+How to configure scaling behaviour will depend on the infrastructure you're using, and the use case you have implemented. For example, when scaling based on CPU usage, this metric will be affected by the size and shape of the data, which transformations and filters are used, and if using script transformations, the content of those scripts. 
+
+We recommend that you instrument your use case, and follow the best practices for the deployment infrastrucutre you're using to find a the scaling configuration which suits your requirements. Note that occasionally releases will improve efficiency, which in the past has had a large impact on metrics which can be used for scaling. So, it is advisable to monitor those metrics after any version update or transformation configuration change, to ensure that scaling behaves as expected.
```

Comment:
```suggestion
We recommend that you instrument your use case, and follow the best practices for the deployment infrastructure you’re using to find a the scaling configuration which suits your requirements. Note that occasionally releases will improve efficiency, which in the past has had a large impact on metrics which can be used for scaling. So, it is advisable to monitor those metrics after any version update or transformation configuration change, to ensure that scaling behaves as expected.
```

I am not sure this advice is practical enough.

> follow the best practices for the deployment infrastructure you’re using to find a the scaling configuration which suits your requirements

That just sounds like a truism, why would not someone follow the best practices? :D
Could we suggest specific ways to set up autoscaling that we recommend and/or use?

Reply:
I agree that it's an empty statement. However I don't have any more specific recommendation to make - there are too many variables in figuring out what scaling for your use case needs to be, it has to be a case by case decision.

I think it's hard to explain the reasoning over text - I struggled to articulate it fully in the email thread. I've removed this line for now, but I'm more than happy to hop on a zoom to understand each other on this if that's helpful. :)